5.6/test/run

```diff
@@ -198,6 +197,7 @@ function try_image_invalid_combinations() {
 
 function run_container_creation_tests() {
   echo "  Testing image entrypoint usage"
+  assert_container_creation_fails "$@"
```

Comment:
We never call run_container_creation_tests with arguments, so `"$@"` here is redundant.